web/styles/modal.css

```diff
@@ -383,3 +383,81 @@
             0 0 8px hsl(206deg 80% 62% / 60%);
     }
 }
+
+#add-poll-modal {
+    /* this height allows 3-4 option rows
+    to fit in without need for scrolling */
+    height: 450px;
+    overflow: hidden;
+
+    .modal__content {
+        flex-grow: 1;
+
+        .simplebar-content {
+            box-sizing: border-box;
+            height: 100%;
+        }
+    }
+
+    #add-poll-form {
+        display: flex;
+        flex-direction: column;
+        overflow: hidden;
+        height: 100%;
+
+        & label {
```

Comment:
Better to attach a class to both this label and the `input` below. 

See, for example, https://chat.zulip.org/#narrow/stream/6-frontend/topic/CSS.20optimization/near/1665704.